testing_and_setup/compass/ocean/hurricane/USDEQU120at30cr10rr2/sandy/config_forward.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0"?>
+<config case="forward">
+	<add_link source="../init/ocean.nc" dest="input.nc"/>
+	<add_link source="../init/graph.info" dest="graph.info"/>
+	<add_link source="../init/atmospheric_forcing.nc" dest="atmospheric_forcing.nc"/>
+	<add_link source="../init/points.nc" dest="points.nc"/>
+
+	<add_executable source="model" dest="ocean_model"/>
+	<add_executable source="metis" dest="metis"/>
```

Comment:
```
┌─[pwolfram][gr-fe1][/lustre/scratch4/turquoise/pwolfram/PR_tests/MPAS-Model_huricanne/testing_and_setup/compass][16:03][±][hurricane_squashed ✗]
└─▪ ./clean_testcase.py -n 67; ./setup_testcase.py -n 67 -f general.config.ocean 
 -- Removed case /lustre/scratch4/turquoise/.mdt1/pwolfram/PR_tests/MPAS-Model_huricanne/testing_and_setup/compass/ocean/hurricane/USDEQU120at30cr10rr2/sandy/init
 WARNING: No runtime definition selected. Using the default of /lustre/scratch4/turquoise/.mdt1/pwolfram/PR_tests/MPAS-Model_huricanne/testing_and_setup/compass/runtime_definitions/mpirun.xml
ERROR: Configuration ocean/hurricane/USDEQU120at30cr10rr2/sandy/config_init.xml requires a definition of metis.
```

Do we need this line?  I believe it can be removed.

Reply:
@xylar, metis was removed from the `general.config.ocean` file but doesn't appear to be in in the conda mpas_tools distribution.  I would argue it should be to remove lines like this from our xml code.

Reply:
We need to remove this and other lines referencing metis because it isn't in `general.config.ocean` anymore.

Reply:
I think this worked for me because I still have metis in my `general.config.ocean` I believe it's part of the conda package now. I'll remove that link.

Reply:
Thanks!

Reply:
> metis was removed from the general.config.ocean file but doesn't appear to be in in the conda mpas_tools distribution. I would argue it should be to remove lines like this from our xml code.

@pwolfram, I think you're still confusing `mpas_tools` with `compass`.  The `compass_3.7` environment, as defined in [README_ocean.md](https://github.com/MPAS-Dev/MPAS-Model/blob/e638d7c721cac925260407b3374973c72ae4ecd8/testing_and_setup/compass/README_ocean.md), is:
```bash
conda create -n compass_py3.7 -c conda-forge python=3.7 \
    geometric_features=0.1.1 mpas_tools=0.0.3 jigsaw=0.9.10 metis pyflann \
    scikit-image basemap pyamg ffmpeg
```
So that's where `metis` comes in, not via `mpas_toools`.

Reply:
Thanks @xylar, I agree and this is obvious-- clearly had end of tiredness yesterday.